Detectors/Vertexing/src/SVertexer.cxx

```diff
@@ -494,9 +518,14 @@ void SVertexer::buildT2V(const o2::globaltracking::RecoContainer& recoData) // a
         }
         continue;
       }
+
+      if (!hasTPC && nITSclu < mSVParams->mITSSAminNclu) {
```

Comment:
I thought you wanted to avoid using these tracks in the cascades only. We don't have a problem with V0 finding timing and rejecting on this level may affect K0s efficiency.